jerry-core/ecma/base/ecma-globals.h

```diff
@@ -204,7 +204,7 @@ typedef enum
 
   ECMA_INTERNAL_PROPERTY_NATIVE_HANDLE, /**< native handle associated with an object */
   ECMA_INTERNAL_PROPERTY_FREE_CALLBACK, /**< object's native free callback */
-
+  ECMA_INTERNAL_PROPERTY_NOT_FOUND_AND_STOP, /**< the property is not found and no more search */
```

Comment:
```
Int8Array.prototype['-1.1']='abc'
>"abc"
var a = new Int8Array(10);
>undefined
a['-1.1']
>undefined
a['-1.1']=5
>5
a.hasOwnProperty('-1.1')
>false
```
'-1.1' will never be the property of typedarray, because it is a ' CanonicalNumeric', but it is not valid index. 
because it is a 'CanonicalNumeric', the spec says it cannot search though the proto chain any more